test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1513,4 +1523,8 @@ static void checkEqualLargeObject(Object largeObject, Object contentObject) thro
 			assertEquals(getObject.invoke(getObjects[i].invoke(largeObject)), contentObject);
 		}
 	}
+	
+	static void checkObject(Object ...objects) {
```

Comment:
- Why is the method called `checkObject`, but it calls `SystemDump()`? And the parameter `objects ` is not used.
- Will we use this method in the future?

Reply:
@llxia 

The purpose of this method is to make it easy to find the objects in DDR when the system core is produced. This method lets one put an object (or many objects) in the argument section of the method frame. When the SystemDump is produced, the object will be in the stack frame and it will be easy to find it in DDR since it will be in the arguments section.  

The alternative approach is to inspect the bytecodes and determine which temp/local slot the object of interest is located in. This approach is more difficult and time consuming then having the objects in known argument slots.

This method is only used for debugging purposes.